assets/src/design-system/components/pill/pillGroup.js

```diff
@@ -28,6 +28,7 @@ const Container = styled.div.attrs({
   role: 'listbox',
 })`
   display: flex;
+  flex: 1;
 `;
 
 function PillGroup({ options, value, onSelect }) {
```

Comment:
I think we need to change the pills slightly, so that the currently focused pill has `z-index:1` to make the outline go above any active neighbor:

![Screen Shot 2021-03-04 at 10 58 16](https://user-images.githubusercontent.com/637548/109991925-c4b66500-7cd8-11eb-8a9e-de220ebb4b44.png)